### PR TITLE
[framework] rewrite of shopsys:phing-targets:sort command

### PR DESCRIPTION
**Pull request by PetrHeinz**
- renamed to `shopsys:phing-config:fix`
- the class was renamed to `PhingConfigFixerCommand`
- `SymfonyStyle` is used for input / output
- return codes are used instead of throwing Exceptions (Exceptions are still thrown for actual errors)
- messages were tweaked for better understandability
- the command also normalizes the whitespace in XML

| Q             | A
| ------------- | ---
|Description, reason for the PR| Usage of shopsys:phing-targets:sort introduced in #1104 was clumsy and caused a lot of hard-to-find errors.
|New feature| No 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| No (the class was not released yet) 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by simara-svatopluk** [comment]
@PetrHeinz sorry, I don't have time until next standup (and if there is anything about API, I cannot guarantee my time on monday)

File: packages/framework/src/Command/SortPhingTargetsCommand.php [hunk 1]
```diff
@@ -31,31 +33,52 @@ protected function configure()
     /**
      * @param \Symfony\Component\Console\Input\InputInterface $input
      * @param \Symfony\Component\Console\Output\OutputInterface $output
+     * @return int
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```
Comment (boris-brtan) [inline packages/framework/src/Command/SortPhingTargetsCommand.php]: so in this case, dont we want to use also `declare(strict_types=1);` ?

Comment (PetrHeinz) [reply packages/framework/src/Command/SortPhingTargetsCommand.php]: You're right, thanks for noticing!

**Comment by boris-brtan** [comment]
TESTS seems OK now

**Pull request merged**